go-controller/pkg/ovn/gateway.go

```diff
@@ -0,0 +1,67 @@
+package ovn
+
+import (
+	"os/exec"
+	"strings"
+
+	"github.com/Sirupsen/logrus"
+)
+
+func (ovn *Controller) getOvnGateways() ([]string, error) {
+	// Return all created gateways.
```

Comment:
When I wrote this for Python, I did not cache the contents of the gateways because I thought people may add new gateways dynamically. I am still torn by that. Ideally when we use OVSDB protocol, we will get notifications when new gateways are added and the cache can be updated.